src/EditorFeatures/CSharpTest/AddBraces/AddBracesFixAllTests.cs

```diff
@@ -1,267 +0,0 @@
-﻿using System.Threading.Tasks;
```

Comment:
There were only three tests here, so I moved them into the main test file for Add Braces.